Initialized datatset with 800 images.
Initialized datatset with 400 images.
self.seg_included: True
last 2208
freeze layers:
**********************************
12
0 torch.Size([1, 96, 112, 112])
1 torch.Size([1, 96, 112, 112])
2 torch.Size([1, 96, 112, 112])
3 torch.Size([1, 96, 56, 56])
4 torch.Size([1, 384, 56, 56])
5 torch.Size([1, 192, 28, 28])
6 torch.Size([1, 768, 28, 28])
7 torch.Size([1, 384, 14, 14])
8 torch.Size([1, 2112, 14, 14])
9 torch.Size([1, 1056, 7, 7])
10 torch.Size([1, 2208, 7, 7])
11 torch.Size([1, 2208, 7, 7])
************************
Down sample at [0, 3, 5, 7, 9]
Number of out channels [96, 384, 768, 2112, 2208]
From_Layer:0 to_Layer:2
From_Layer:3 to_Layer:4
From_Layer:5 to_Layer:6
From_Layer:7 to_Layer:8
From_Layer:9 to_End
********************
[32m[I 2024-01-19 13:15:33,281][39m A new study created in RDB with name: Study_database
[33m[W 2024-01-19 13:15:34,227][39m Trial 0 failed with parameters: {'deconv_layers_3_no_channels': 0.125, 'deconv_layers_5_no_channels': 0.0625, 'atrous_conv_layers_2_no_channels': 0.25, 'atrous_conv_layers_3_no_channels': 0.25, 'max_min_expan_layers_no_channels': 0.125} because of the following error: ValueError('out_channels must be divisible by groups').
Traceback (most recent call last):
  File "/home/pupil/rmf3mc/.conda/envs/UnetCRF2/lib/python3.10/site-packages/optuna/study/_optimize.py", line 200, in _run_trial
    value_or_values = func(trial)
  File "/home/pupil/rmf3mc/Documents/ModelProposing/MGANet/optuna_Final/Train.py", line 247, in objective
    model=MMANET(trial,num_classes=num_classes,MANet=MANet,MMANet=MMANet,mask_guided=mask_guided,seg_included=seg_ild,freeze_all=freeze_all,Unet=args.unet,transform_to=args.transfer_to)
  File "/home/pupil/rmf3mc/Documents/ModelProposing/MGANet/optuna_Final/backboneModels.py", line 138, in __init__
    self.tranformers[str(i)]=nn.Conv2d(sum_of_current_channels,sum_of_current_channels*transform_to,kernel_size=1)
  File "/home/pupil/rmf3mc/.conda/envs/UnetCRF2/lib/python3.10/site-packages/torch/nn/modules/conv.py", line 444, in __init__
    super(Conv2d, self).__init__(
  File "/home/pupil/rmf3mc/.conda/envs/UnetCRF2/lib/python3.10/site-packages/torch/nn/modules/conv.py", line 86, in __init__
    raise ValueError('out_channels must be divisible by groups')
ValueError: out_channels must be divisible by groups
[33m[W 2024-01-19 13:15:34,229][39m Trial 0 failed with value None.
Traceback (most recent call last):
  File "/home/pupil/rmf3mc/Documents/ModelProposing/MGANet/optuna_Final/Train.py", line 509, in <module>
    study.optimize(objective, n_trials=2, timeout=None,  callbacks=[print_callback])
  File "/home/pupil/rmf3mc/.conda/envs/UnetCRF2/lib/python3.10/site-packages/optuna/study/study.py", line 451, in optimize
    _optimize(
  File "/home/pupil/rmf3mc/.conda/envs/UnetCRF2/lib/python3.10/site-packages/optuna/study/_optimize.py", line 66, in _optimize
    _optimize_sequential(
  File "/home/pupil/rmf3mc/.conda/envs/UnetCRF2/lib/python3.10/site-packages/optuna/study/_optimize.py", line 163, in _optimize_sequential
    frozen_trial = _run_trial(study, func, catch)
  File "/home/pupil/rmf3mc/.conda/envs/UnetCRF2/lib/python3.10/site-packages/optuna/study/_optimize.py", line 251, in _run_trial
    raise func_err
  File "/home/pupil/rmf3mc/.conda/envs/UnetCRF2/lib/python3.10/site-packages/optuna/study/_optimize.py", line 200, in _run_trial
    value_or_values = func(trial)
  File "/home/pupil/rmf3mc/Documents/ModelProposing/MGANet/optuna_Final/Train.py", line 247, in objective
    model=MMANET(trial,num_classes=num_classes,MANet=MANet,MMANet=MMANet,mask_guided=mask_guided,seg_included=seg_ild,freeze_all=freeze_all,Unet=args.unet,transform_to=args.transfer_to)
  File "/home/pupil/rmf3mc/Documents/ModelProposing/MGANet/optuna_Final/backboneModels.py", line 138, in __init__
    self.tranformers[str(i)]=nn.Conv2d(sum_of_current_channels,sum_of_current_channels*transform_to,kernel_size=1)
  File "/home/pupil/rmf3mc/.conda/envs/UnetCRF2/lib/python3.10/site-packages/torch/nn/modules/conv.py", line 444, in __init__
    super(Conv2d, self).__init__(
  File "/home/pupil/rmf3mc/.conda/envs/UnetCRF2/lib/python3.10/site-packages/torch/nn/modules/conv.py", line 86, in __init__
    raise ValueError('out_channels must be divisible by groups')
ValueError: out_channels must be divisible by groups
0.125 0.0625 0.25 0.25 0.125